### PR TITLE
done homework

### PR DESCRIPTION
**Pull request by Wolframcheg**

### CODE REVIEW

File: src/Transport/AbstractTransport.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    public abstract function getName();
+    public abstract function setName($name);
+    public abstract function getModel();
+    public abstract function setModel($model);
+    public abstract function getMaxSpeed();
+    public abstract function setMaxSpeed($maxSpeed);
+    public abstract function printAllInfo();
+
+
+    public function __set($name, $value)
```
Comment (ahonymous) [inline src/Transport/AbstractTransport.php:15]: Don`t use magick __set and __get.

Comment (Wolframcheg) [inline src/Transport/AbstractTransport.php:15]: done

File: src/Transport/AbstractTransport.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?php
+namespace Transport;
+
+abstract class AbstractTransport
```
Comment (ahonymous) [inline src/Transport/AbstractTransport.php:4]: I thick, that it`s not class - it`s interface

Comment (Wolframcheg) [inline src/Transport/AbstractTransport.php:4]: ok) I transformed the Abstract class to the Interface

**Pull request merged**